.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,72 @@
+<!--  All Submissions -->
+
+
+## Community Note
+<!-- Please leave the community note as is. -->
+* Please vote on this PR by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original PR to help the community and maintainers prioritize for review
+* Please do not leave "+1" or "me too" comments, they generate extra noise for PR followers and do not help prioritize for review
+
+
+## PR Checklist
+
+- [ ] I have followed the guidelines in our [Contributing Documentation](../contributing/README.md).
+- [ ] I have checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change.
+- [ ] I have used a meaningful PR description to help maintainers and other users understand this change and help prevent duplicate work.
+Example:
+“`resource_name_here` - description of change e.g. adding property `new_property_name_here`”
+- [ ] I have checked if my changes close any open issues. If so please include appropriate [closing keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) below.
+
+
+<!-- You can erase any parts of this template below this point that are not applicable to your Pull Request. -->
+
+
+## New Feature Submissions
+
+- [ ] My submission includes Test coverage as described in the [Contribution Guide](../contributing/topics/guide-new-resource.md) and the tests pass. (if this is not possible for any reason, please include details of why below)
+
+
+## Changes to existing Resource / Data Source
+
+- [ ] I have added an explanation of what my changes do and why I'd like you to include them (This may be covered by linking to an issue above, but may benefit from additional explanation).
+- [ ] I have written new tests for my resource or datasource changes.
+- [ ] I have successfully run tests with my changes locally. If not, please provide details on testing challenges that prevented you running the tests.
+
```

Comment:
State migrations can be opened as standalone PRs to fix bugs, or in combination with other changes that could classify more as enhancements to an existing resource, so I'm not sure if it wants it's own header or could just be tagged on here as another check box.
```suggestion
## Changes to existing Resource / Data Source

- [ ] I have added an explanation of what my changes do and why I'd like you to include them (This may be covered by linking to an issue above, but may benefit from additional explanation).
- [ ] I have written new tests for my resource or datasource changes.
- [ ] I have successfully run tests with my changes locally. If not, please provide details on testing challenges that prevented you running the tests.
- [ ] I have successfully run tests with my changes locally. If not, please provide details on testing challenges that prevented you running the tests.
- [ ] (For changes that include a state migration only). I have manually tested the migration path between relevant versions of the provider.
```

Reply:
I will add this and I am going to bold **state migration only** instead of moving it to its own header. We can try this for now, and if in the future we're not getting the interaction we are aiming for, we can update the template.